### PR TITLE
data : priyanshu-11000

### PR DESCRIPTION
**Pull request by priyanshu-11000**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4450"></a>

### CODE REVIEW

File: data/priyanshu-11000.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "name": "Priyanshu Kumar Singh",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "BTech. 3rd year Student | CSE NITDGP'24 | Open Source Enthusiast | Blockchain Enthusiast | No-Code tools",
+    "avatar": "https://bit.ly/3JDlnoa",
```
Comment (Panquesito7) [inline data/priyanshu-11000.json:6]: The `avatar` field is no longer required as of a new update.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

File: data/priyanshu-11000.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "BTech. 3rd year Student | CSE NITDGP'24 | Open Source Enthusiast | Blockchain Enthusiast | No-Code tools",
+    "tags": [
+      "Open Source",
+      "C",
+      "Python",
+      "C++",
+      "JAVA",
+      "HTML/CSS"
```
Comment (priyankarpal) [inline data/priyanshu-11000.json:12]: You can add separate Tags such as "HTML", "CSS"

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyanshu-11000 — COMMENTED** [review]
Added some more tags in the file.

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Review by amandamartin-dev — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 


<p align="center"></p>

**Pull request merged**